device-types/Dell/PowerEdge_R240.yaml

```diff
@@ -0,0 +1,26 @@
+---
+manufacturer: Dell
+model: PowerEdge R240
+slug: dell_poweredge_r240
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+weight: 12.2
+weight_unit: kg
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: VGA
+    type: other
+power-ports:
```

Comment:
Is this PSU modular? If so, this should be modelled in a `module-bays`, and then the applicable PSU should be added to the module-types folder